geode-serialization/src/main/java/org/apache/geode/internal/serialization/StaticSerialization.java

```diff
@@ -494,4 +494,64 @@ public static void writePrimitiveClass(Class c, DataOutput out) throws IOExcepti
             String.format("unexpected typeCode: %s", typeCode));
     }
   }
+
```

Comment:
ah I see why you moved this logic here from `InternalDataSerializer`: the latter is in `geode-core` and our new `MemberIdentifierImpl` is down in `geode-membership` and can't depend on stuff in core ✓